pwa/src/templates/schemasDetailTemplate/SchemasDetailTemplate.tsx

```diff
@@ -110,7 +111,18 @@ export const SchemasDetailTemplate: React.FC<SchemasDetailPageProps> = ({ schema
               <FontAwesomeIcon icon={faPlus} /> {t("Add Object")}
             </Button>
 
-            {getObjectsFromEntity.isSuccess && <ObjectsTable objects={getObjectsFromEntity.data} {...{ schemaId }} />}
+            {getObjectsFromEntity.isSuccess && (
+              <ObjectsTable
+                objects={getObjectsFromEntity.data.results}
+                pagination={{
+                  totalPages: getObjectsFromEntity.data.pages,
+                  currentPage: currentObjectsPage,
+                  changePage: setCurrentObjectsPage,
+                }}
+                {...{ schemaId }}
+              />
+            )}
```

Comment:
I think we should add some spacing between the table and the pagination:

![image](https://user-images.githubusercontent.com/66552932/220945941-6a923128-041d-471f-81dc-c61c36dd01e3.png)